### PR TITLE
Add Modal and Toast component

### PR DESCRIPTION
**Pull request by VLeads**
Added Modal Component.
Added Toast Component.

### CODE REVIEW

File: styles/lib/_modal.css [hunk 1]
```diff
@@ -0,0 +1,55 @@
+.modal{
+    display: none;
+    position: fixed;
+    z-index: 100;
```
Comment (vrinda-mahajan) [inline styles/lib/_modal.css:4]: Avoid giving such high values to the z-index. You can instead make it 2 or 3.

File: styles/lib/_toast.css [hunk 2]
```diff
@@ -0,0 +1,48 @@
+ #toast{
+    min-width: 28rem;
+    margin-left: -125px;
+    background-color: #333;
+    color: var(--white);
+    /* text-align: center; */
```
Comment (vrinda-mahajan) [inline styles/lib/_toast.css:6]: You can remove this comment also.

**Review by vrinda-mahajan — CHANGES_REQUESTED** [review]

**Review by ParthParolekar — APPROVED** [review]
Everything looks good to me

File: styles/lib/_modal.css [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    padding-top: 12rem;
+    left: 0;
+    top: 0;
+    width: 100%;
+    height: 100%;
+    overflow: auto;
+    background-color: rgba(0,0,0,0.4);
```
Comment (deekshithmd) [inline styles/lib/_modal.css:11]: try to use color variables

File: styles/lib/_modal.css [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+    -webkit-animation-duration: 0.4s;
+    animation-name: animatetop;
+    animation-duration: 0.4s
+}
+
+
+.modal-header{
+    padding: 1.2rem 2rem;
+    border-bottom: thin solid var(--grey);
+    margin: 0.8rem 0;
```
Comment (deekshithmd) [inline styles/lib/_modal.css:30]: for values less than 10px try to use px instead of rem

**Review by deekshithmd — APPROVED** [review]

File: styles/lib/_modal.css (hunk 1 above)
Comment (VLeads) [reply styles/lib/_modal.css:4]: I gave it because I want modal on top of everything while appearing.

File: styles/lib/_toast.css (hunk 2 above)
Comment (VLeads) [reply styles/lib/_toast.css:6]: Thanks, I will remove it.

File: styles/lib/_modal.css (hunk 3 above)
Comment (VLeads) [reply styles/lib/_modal.css:11]: I will not use this color elsewhere. That's why I haven't used variable name.

File: styles/lib/_modal.css (hunk 4 above)
Comment (VLeads) [reply styles/lib/_modal.css:30]: Ohk, I'll update this.

**Comment by VLeads** [comment]
Merging the branch after review by peers.

**Pull request merged**